source/Government.cpp

```diff
@@ -136,6 +136,8 @@ void Government::Load(const DataNode &node)
 			raidFleet = GameData::Fleets().Get(child.Token(1));
 		else if(child.Token(0) == "provoked on scan")
 			provokedOnScan = true;
+		else if(child.Token(0) == "restricted")
+			restricted.Load(child);
 		else
```

Comment:
Support "add" and "remove" instructions, too.

Reply:
> Support "add" and "remove" instructions, too.

When you say remove do you mean removing all restrictions or remove only what you specify? 
Maybe remove all if nothing is specified, else just remove what is asked?

I'm asking this because I will have to add that to the locationFilter. ("add" its already the default behaviour to not overwrite the node entirely but just add new things, but maybe default should be made to overwrite?)

Reply:
If a plugin modifies an existing government and does not specify "add," then the behavior should be to replace any existing travel restrictions with the provided ones. Only if explicitly opting in to the existing travel restrictions via "add" should their conditions get merged with whatever was previously parsed in a previous definition.